six/two/two.cpp

```diff
@@ -472,7 +456,8 @@ const char *Two::runCheck(SixPyObject *check)
 
     // result will be eventually returned as a copy and the corresponding Python
     // string decref'ed, caller will be responsible for memory deallocation.
-    char *ret, *ret_copy = NULL;
```

Comment:
this doesn't do what you might expect it does :)

Reply:
sneaky uninitialized `ret` ;)